docker/multi-process/scripts/init

```diff
@@ -5,6 +5,15 @@ export LC_ALL=en_US.UTF-8
 
 cd /app
 
+# Do we have a Rails secret yet?
+# If not, generate one, but persist it for the container's lifetime.
+if grep -q ^APP_SECRET_TOKEN=REPLACE_ME_NOW .env.example; then
```

Comment:
That's precisely what the grep condition does: the block only executes if `APP_SECRET_TOKEN` is set to the git distribution default value (`REPLACE_ME_NOW!`). If it's set to anything else, the grep fails and the `if` block is skipped.

It does leave the `!` in place as a script simplification, since that is a metacharacter to bash.